Docs/Whats_New.md

```diff
@@ -18,14 +18,39 @@ MoltenVK 1.4.1
 
 Released TBD
 
-- .
-- New state tracker implementation.
-- New descriptor set/pool implementation.
-	- The new implementation is less forgiving of applications that don't properly manage the lifetime of their descriptors
-	  (and bind them after the objects they point to have been destroyed).
-	  Use `MVK_CONFIG_LIVE_CHECK_ALL_RESOURCES=1` to revert any crashes this might cause, or mark potentially-destroyed descriptors
-	  with `VK_DESCRIPTOR_BINDING_PARTIALLY_BOUND_BIT`.
+- Improved occlusion query support:
+  - Add support for occlusion queries across multiple renderpasses.
+  - Improve performance by accumulating occlusion quieries in parallel, 
+    and using untract buffers to avoid encoder serialization.
```

Comment:
"untract"?? I'm pretty sure that's not a word. I'd suggest a change, but it isn't clear what word was actually meant here.

Reply:
Thanks. Fixed.